VorpalEngine/include/vorpal/utility/asset/SimpleStorage.hpp

```diff
@@ -16,54 +17,54 @@
 
 namespace vp
 {
-    namespace utility
-    {
-        namespace asset
-        {
-            /** @brief  Controls non-thread safe asset access
-             *
-             *  Provides synchronous method to request assets
-             *
-             *  @todo   cleaning up idle Handler object
-             */
-            class SimpleStorage : public templates::Singleton<SimpleStorage>
-            {
-            public:
-                typedef std::string KeyType;
+namespace utility
+{
+namespace asset
+{
+/** @brief  Controls non-thread safe asset access
+  *
+ *  Provides synchronous method to request assets
+ *
+ *  @todo   cleaning up idle Handler object
+ */
+class SimpleStorage : public templates::Singleton<SimpleStorage>
+{
+   public:
+    typedef std::string KeyType;
```

Comment:
~Looks like it completely fucked up in this file~
I don't like how it indents access levels